project/app/src/main/res/values-es/strings.xml

```diff
@@ -0,0 +1,208 @@
+    <string name="successCopyCertificate">Certificado copiado al almacenamiento interno</string>
+    <string name="preferencesCategoryNotificationOngoing">Ongoing notification</string>
+    <string name="preferencesCategoryNotificationBackground">Notificaciones en segundo plano</string>
+    <string name="preferencesCategoryAdvancedServices">Servicios</string>
+    <string name="preferencesCategoryAdvancedLocator">Locator</string>
+    <string name="preferencesCategoryAdvancedEncryption">Encriptación</string>
+    <string name="preferencesCategoryAdvancedMisc">Miscelánea</string>
+    <string name="contactDetailsTrackerId" translatable="false">Tracker ID</string>
+    <string name="waypointRemoved">Región eliminada</string>
+
+    <string name="exportWaypointsToEndpoint">Publicar Waypoints</string>
+    <string name="preferencesExportFailed">Fallo al exportar las preferencias</string>
+    <string name="preferencesExportSuccess">Preferencias exportadas</string>
+    <string name="geofenceInactive">Geofence desactivado</string>
+    <string name="geofenceActive">Geofence activado</string>
+
+
+    <string name="welcomeFixIssue">Fix</string>
+    <string name="play_services_now_available">Google Play Services disponibles</string>
+    <string name="play_services_not_available">Google Play Services no disponibles</string>
+    <string name="preferencesUrl" translatable="false">URL</string>
+    <string name="mode_http_private_label">HTTP privada</string>
+
+
+    <string name="preferencesKeyModeId" translatable="false">mode</string>
+    <string name="preferencesKeyHost" translatable="false">host</string>
+    <string name="preferencesKeyIdentification" translatable="false">identification</string>
+    <string name="preferencesKeySecurity" translatable="false">security</string>
+    <string name="preferencesKeyParameters" translatable="false">parameters</string>
+    <string name="transitionEntering">entra</string>
+    <string name="transitionLeaving">sale</string>
+    <string name="errorPreferencesImportFailed">No se pudo importar</string>
+    <string name="welcome_play_heading">Google Play Services</string>
+    <string name="welcome_play_description">Owntracks necesita Google Play Services</string>
+    <string name="icon_description_warning">Alerta</string>
+    <string name="play_services_not_available_general">Google Play Services no disponibles</string>
+    <string name="welcome_heading">Bienvenido</string>
+    <string name="welcome_description">Los siguientes pasos te ayudarán a comenzar con Owntracks</string>
+    <string name="mode_heading">Modo</string>
+    <string name="mode_description">Owntracks puede funcionar en uno de los siguientes modos. Elige el modo que mejor se adapte a ti.</string>
+    <string name="mqtt_public_description">Prueba Owntracks y comparte tu ubicación de manera anónima con otros usuarios</string>
```

Comment:
@binarybucks I think this, too, is superflous

Reply:
* `welcome_description`, `mode_heading`, is still used
* `mode_description`, `mqtt_public_description` and `http_public_description`can be removed.